draft-ietf-quic-applicability.md

```diff
@@ -898,7 +905,9 @@ See the security considerations in {{QUIC}} and {{!QUIC-TLS}}; the security
 considerations for the underlying transport protocol are relevant for
 applications using QUIC, as well. Considerations on linkability, replay attacks,
 and randomness discussed in {{!QUIC-TLS}} should be taken into account when
-deploying and using QUIC.
+deploying and using QUIC. Further, migration to an new address exposes
```

Comment:
This new point is worth a new paragraph.

> Using migration reveals to the server that the same client controls multiple addresses.

Reply:
```suggestion
deploying and using QUIC.

Further, migration to an new address exposes
```